react/Hidden/Hidden.less

```diff
@@ -0,0 +1,15 @@
+// Todo: add responsive breakpoints support later in the same component
+
+.print {
+  @media print {
+    display: none;
+  }
+}
+
+.screen {
+  display: none;
+
+  @media print {
```

Comment:
Oh right, yeah, sorry.

Could this be simplified to `@media screen { display: none; }` like the print styles?